app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/subscription/Sub02Controller.scala

```diff
@@ -59,22 +59,38 @@ class Sub02Controller @Inject() (
       }
   }
 
-  private def renderPageWithName(service: Service)(implicit request: Request[_]): Future[Result] =
+  private def renderPageWithName(service: Service)(implicit request: Request[_]): Future[Result] = {
+    val subscriptionTo = s"cds.subscription.outcomes.steps.next.${service.code}"
     for {
       sub02Outcome <- sessionCache.sub02Outcome
       _            <- sessionCache.remove
       _            <- sessionCache.saveSub02Outcome(sub02Outcome)
     } yield Ok(
-      migrationSuccessView(sub02Outcome.eori, sub02Outcome.fullName, sub02Outcome.processedDate, service)
+      migrationSuccessView(
+        sub02Outcome.eori,
+        sub02Outcome.fullName,
+        sub02Outcome.processedDate,
+        subscriptionTo,
+        service
+      )
     ).withSession(newUserSession)
+  }
 
-  private def renderPageWithNameRow(service: Service)(implicit request: Request[_]): Future[Result] =
+  private def renderPageWithNameRow(service: Service)(implicit request: Request[_]): Future[Result] = {
+    val subscriptionTo = s"cds.subscription.outcomes.steps.next.${service.code}"
```

Comment:
Let's avoid the magic strings - this can either be a def you can feed from a Constants object or some other implementation.